src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/ApprenticeshipsController.cs

```diff
@@ -25,18 +28,44 @@ public ApprenticeshipsController(IMediator mediator, ILogger<ApprenticeshipsCont
 
         [HttpGet]
         [Route("{providerId}")]
-        public async Task<IActionResult> GetApprenticeships(uint providerId)
+        public async Task<IActionResult> GetApprenticeships(uint providerId, [FromQuery]int pageNumber = 0, [FromQuery]int pageItemCount = 0, [FromQuery]string sortField = "", [FromQuery]bool reverseSort = false)
```

Comment:
I mentioned on slack the issue of providerId appearing in the route. Currently your route looks like this:

`/api/apprenticeships/123`

What this says to me is, that this resource is an apprenticeship (it's in the apprenticeships collection) with an Id of 123, which is not the intention I assume.

Additionally, how will this endpoint be usable by the employer as it stands?

I recommended adding providerId to the query string and modelling the request like so, as we did in the Cohort controller:

```  
public class GetCohortsRequest
{
   public long? AccountId { get; set; }
   public long? ProviderId { get; set; }
}
```

Reply:
sure no problem i'll change that now